charts/illumidesk/values.yaml

```diff
@@ -300,6 +300,18 @@ jupyterhub:
       type: ClusterIP
   # Single user server
   singleuser:
+    # Default CPU Limits and Gurantees
+    cpu:
+      # Guranteed CPU allocation
+      guarantee: 0.2
```

Comment:
I don't think 0.2 CPU is enough. Let's bump this to 0.5. Using the ratio mentioned above, we should set CPU to `0.5` since memory is set to `1Gb` by default.

Reply:
Fixed single user resources